packages/server/src/routes/authenticate-admin.js

```diff
@@ -0,0 +1,21 @@
+import Models from '../models'
+import HttpError from '../errors/http-error'
+import asyncMiddleware from '../helpers/async-middleware'
+
+const { Admin } = Models
+
+const authenticate = route => async (request, response, next) => {
+  const { session, cookies } = request
+
+  if (!session.modelId || !session.modelType) {
+    // Check if user's cookie is still saved in browser and user is not set, then automatically log the user out
+    // This usually happens when you stop your express server after login, your cookie still remains saved in the browser
+    if (cookies && cookies.aragonCourtSessionID) response.clearCookie('aragonCourtSessionID')
```

Comment:
I don't think this changes much. If there is no valid cookie in the backend, the server will just ignore it and automatically respond with a new `set-cookie` header

Reply:
It would be like forcing that header, not sure if the server does that automatically, but if it does then we can remove this line

Reply:
that's quite standard for session cookies, browser always syncs with whatever is inside `set-cookie`

Reply:
Also it's going to mess up the request if for example you want to authenticate (create cookie) but you also deleting a cookie in the same request. better to remove this.